### ISSUE TITLE
Feature proposal: Add tab to show sources, which are linked to a shared places

### ISSUE THREAD

**Issue by Jefferson49** [vesta-webtrees-2-custom-modules/vesta_shared_places#156]
Since I am adding shared places to al lot of my sources, it would be helpful to show an additional tab for sources linked to a shared place. 

Example Gedcom for a source, which is linked to a shared place:
```
0 @S10207@ SOUR
1 TITL Standesamt Fehrbellin, Geburtsregister 1874-1878
1 DATA
2 EVEN BIRT
3 DATE FROM 1874 TO 1878
3 PLAC Fehrbellin, Landkreis Ostprignitz-Ruppin, Brandenburg, DEU
4 _LOC @P232@
```

![Sources with Shared Place](https://user-images.githubusercontent.com/81484983/235287104-b9ac8ab3-88a0-46dd-85b4-cb8a3d4eb000.JPG)

**Comment by ric2016** [comment]
I think the main reason this isn't already available is that these links aren't available internally for the Shared Places module setting 'Additionally link shared places via name' (webtrees doesn't add entries to the 'placelinks' table for sources).

It should be possible easily to support them if the source directly links to a _LOC though: These links are already stored by webtrees in the 'link' table.

**Comment by ric2016** [comment]
Implemented in Release 2.1.17.0.0 (for direct links only).

**Comment by Jefferson49** [comment]
Danke fürs Einbauen des neuen Features! Sieht bei mir sehr gut aus. Test verlieft mit vesta.2.1.17.0.1 erfolgreich.
